vpr/src/pack/cluster.cpp

```diff
@@ -300,6 +300,11 @@ void add_cluster_molecule_candidates_by_highfanout_connectivity(t_pb* cur_pb,
                                                                 const std::multimap<AtomBlockId, t_pack_molecule*>& atom_molecules,
                                                                 const int feasible_block_array_size);
 
+void add_cluster_molecule_candidates_by_attraction_group(t_pb* cur_pb,
+                                                         t_cluster_placement_stats* cluster_placement_stats_ptr,
+                                                         const std::multimap<AtomBlockId, t_pack_molecule*>& atom_molecules,
+                                                         const int feasible_block_array_size);
```

Comment:
the feasible block array size could go into t_pb_stats with the actual array

Reply:
feasible_block_array_size is the packer option that specifies the max size of the array. It is different from the number of feasible blocks currently in the array, which is already part of t_pb_stats